source/reference/api/start.txt

```diff
@@ -25,6 +25,8 @@ Requirement
 To use the ``start`` endpoint, ``mongosync`` must be in the ``IDLE``
 state.
 
+.. include:: /includes/fact-write-blocking-requirement.rst
```

Comment:
[suggestion]

Should this section be titled "Requirements" since there are two requirements?

Reply:
Done